Source/pack.cpp

```diff
@@ -106,12 +106,12 @@ void PackPlayer(PkPlayerStruct *pPack, const PlayerStruct &player, bool manashie
 	pPack->pDifficulty = SDL_SwapLE32(player.pDifficulty);
 	pPack->pDamAcFlags = SDL_SwapLE32(player.pDamAcFlags);
 	pPack->pDiabloKillLevel = SDL_SwapLE32(player.pDiabloKillLevel);
-	pPack->bIsHellfire = gbIsHellfire;
+	pPack->bIsHellfire = gbIsHellfire ? 1 : 0;

```

Comment:
`bIsHellfire` is always treated as a bool, but due to the way `PkPlayerStruct` is used it needs to have a known width type. Adding a comment to `PkPlayerStruct::bIsHellfire` documenting this would be great.